Makefile

```diff
@@ -47,8 +47,14 @@ GDB     = $(CROSS_COMPILE)gdb
 NRFUTIL = adafruit-nrfutil
 NRFJPROG = nrfjprog
 
-MK = mkdir -p
-RM = rm -rf
+# Set make directory command, Windows tries to create a directory named "-p" if that flag is there.
+ifneq ($(OS), Windows_NT)
+MK := mkdir -p

```

Comment:
look good, though why you use `:=` instead of `=`, if there is not much of difference, we should keep it as `=`

Reply:
I agree completely. It must have been from when I was tinkering to figure out exactly what was making it upset and I just forgot to change it back!